InterviewFlashCards/QueryManager.swift

```diff
@@ -13,40 +13,37 @@ enum RequestType: Int {
     case RequestTypeiOS = 1
```

Comment:
in swift you don't need to prefix your enum case with the enum name. So `case IOS`, case `DataStructures` etc